packages/design-system/src/components/input-field.tsx

```diff
@@ -31,7 +31,7 @@ export const inputFieldColors = ["placeholder", "set", "error"] as const;
 
 const inputStyle = css({
   all: "unset",
-  ...textVariants.regular,
+  ...textVariants.mono,
```

Comment:
can't do that here, the input field is used everywhere, you need to use a CSS prop in specific places where you need that font

Reply:
alternative would be to add a variant "text" and the value would be textVariants, similar to how  Text component has 
```js
export const textStyle = css({
  margin: 0, // in case it's used with <p>
  WebkitFontSmoothing: "antialiased",
  variants: {
    variant: textVariants,
    color: {
```

but in this case it would be 

```js
variants: {
  text: textVariants
}
```

Reply:
I think a text variant is a great idea @istarkov wdyt?

Reply:
@kof sure I'll fix that

Reply:
Hi @kof I have been trying fixing it 
Changes input as you said but somehow its not changing
<img width="281" alt="Screenshot 2023-10-31 224828" src="https://github.com/webstudio-is/webstudio/assets/88244542/3cd6192f-7614-42a5-a6f1-978aff5196f4">
<img width="540" alt="Screenshot 2023-10-31 224908" src="https://github.com/webstudio-is/webstudio/assets/88244542/558ae477-7af2-4d38-ba30-dcb65f66c05f">